examples/drivenet/drivenet_train.ipynb

```diff
@@ -0,0 +1,321 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Introduction to DriveNet\n",
+    "\n",
+    "DriveNet is a DNN able to drive an agent along a specific route. Among others, DriveNet can do lane-following while respecting traffic lights.\n",
```

Comment:
Just a though: could extend this a bit? Maybe one or two more sentences what we want to do (drive the car, using data-driven methods compared to expert planner, use largs amounts of data etc ... )